frontend/src/components/TestScraperForm/TestScraperFormContainer.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import './App.css';
+import { connect } from 'react-redux';
+import { sendURL } from '../../store/actions';
+import TestScraperForm from './TestScraperForm';
+
+class App extends Component {
+	state = {
+		inputData: {
+			url: ''
+		}
+	};
+
+	componentDidMount() {
+		//When mounted, run the sendURL action which calls the API
+		this.props.sendURL();
```

Comment:
Invoke sendURL in componentDidMount? This will try sending a blank input to the backend?

Reply:
woops, this should be fetchPages, good catch